packages/docs/src/locales/en/en.json

```diff
@@ -2381,7 +2381,7 @@
       },
       "allowCreate": {
```

Comment:
Single select shouldn't support array as value. We can throw an error or warn at least.

![image](https://user-images.githubusercontent.com/5394573/124175130-e15f7f80-dab5-11eb-96e0-27db8343dc0c.png)

Reply:
Yeah, thanks it's related to all single selects.

There is no check for this.

I'll fix it.